app/models/customer.py

```diff
@@ -1,4 +1,18 @@
 from app import db
 
 class Customer(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    # videos_checked_out_count = db.Column(db.Integer, default=0, nullable=False)
```

Comment:
I would actually add this column here! You could use it to calculate inventory after someone checks in and checks out a video